src/mantine-demos/tsconfig.build.json

```diff
@@ -34,7 +34,8 @@
       "@mantine/nprogress": ["../mantine-nprogress/src"],
       "@mantine/labs": ["../mantine-labs/src"],
       "@mantine/tiptap": ["../mantine-tiptap/src"],
-      "@mantine/rte": ["../mantine-rte/src"]
+      "@mantine/rte": ["../mantine-rte/src"],
+      "@mantine/ds": ["../mantine-ds/src"]
```

Comment:
tsconfig files should not be changed.

Reply:
when i build project at latest node LTS(v16.17.0) firstly, `tsc --project tsconfig.build.json` will throw error with `@mantine/ds`. So i think `@mantine/ds`'s path should be configured

Reply:
Works fine for me with current implementation, i'd rather not change that.